cob_monitoring/src/battery_monitor.py

```diff
@@ -21,10 +21,12 @@
 import actionlib
 
 from std_msgs.msg import ColorRGBA
-from cob_msgs.msg import PowerState
+from cob_msgs.msg import PowerState, HornMode
+from cob_srvs.srv import SetHornMode, SetHornModeRequest
```

Comment:
add `cob_srvs` as `exec_depend` to `package.xml`

Reply:
Thanks, done!